dls-controls/malcolmjs
# Feature/combobox behaviour #312

## Description

- Changed combobox behaviour so it defaults to first element and displays the given value even if it is not in its choices (in which case the value is not selectable).
- Added transition effect for when child panel becomes parent
- Changed buttons in info panes which launch new views to all launch a new full size window

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/gui/PANDA/layout/SEQ1
- click on the edit button on the table attribute in the child pane and see the transition
- click the popout button on the parent pane and check that clicking the edit button in the new window launches another new, full size window

## Agile board tracking

connect to #312 
closes #312 
connect to #422 
closes #422
